golang/channel.go

```diff
@@ -21,12 +21,23 @@ package tchannel
 // THE SOFTWARE.
 
 import (
+	"errors"
 	"fmt"
-	"golang.org/x/net/context"
 	"net"
 	"os"
 	"path/filepath"
+	"sync"
 	"time"
+
+	"golang.org/x/net/context"
+)
+
+var (
+	errAlreadyListening = errors.New("channel already listening")
```

Comment:
Don't you want this to be a publicly accessible value so that users can match on it?